package.json

```diff
@@ -27,7 +27,9 @@
   "license": "SEE LICENSE IN https://tc39.es/ecma262/#sec-copyright-and-software-license",
   "homepage": "https://tc39.es/ecma262/",
   "dependencies": {
-    "ecmarkup": "9.6.3"
+    "ecmarkup": "9.6.3",
+    "cpy-cli": "*",
```

Comment:
* is dangerous and should never be used in a package.json, ever anywhere.

Reply:
Fixed. I understand that I got too optimistic about forward compatibility even for features that seem basic.